core/storage/user/gae_models.py

```diff
@@ -2366,6 +2366,8 @@ class UserContributionRightsModel(base_models.BaseModel):
     can_review_voiceover_for_language_codes = (
         datastore_services.StringProperty(repeated=True, indexed=True))
     can_review_questions = datastore_services.BooleanProperty(indexed=True)
+    can_submit_questions = datastore_services.BooleanProperty(
+        default=False, indexed=True)
```

Comment:
Why isn't can_review_questions also defaulted? I'm wondering if we can rely on the domain object to always provide a value when populating this fields as we presumably do for can_review_questions. (It seems a bit odd that these two aren't handled the same way).

Reply:
Ah, yes you're right. This does need to be defaulted to properly be handled for existing instances of the model.